Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1744,6 +1804,18 @@
 /* Remove metadata section header */
 "SETTINGS_REMOVE_METADATA_TITLE" = "Metadata";
 
+/* Label for the 'screen lock activity timeout' setting of the privacy settings. */
+"SETTINGS_SCREEN_LOCK_ACTIVITY_TIMEOUT" = "Screen Lock Timeout";
+
+/* Footer for the 'screen lock' section of the privacy settings. */
+"SETTINGS_SCREEN_LOCK_SECTION_FOOTER" = "Lock Signal access with iOS Touch ID or Face ID.";
```

Comment:
Lock Signal when it hasn't been used recently. Unlock it with Touch ID or Face ID (or pass code).

Reply:
Done.